tests/utils/test_embeddings.py

```diff
@@ -241,64 +252,65 @@ def testSubgraphEmbeddingFeasibilityFilter(self):
         )
         # Check tetrahedron cannot be embedded on a graph with a triangle + 0,1,2 connectivity nodes.
 
-    def testRasterBreadthSubgraphUpperBound(self):
+    def testGraphRowsSubgraphUpperBound(self):
```

Comment:
snake case for this and other test functions

Reply:
Yep, its Classes that should have Camel case, good catch